### PR TITLE
Improving json utilities

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

**Review by jcotela — APPROVED** [review]
I'm approving, since these are positive changes, but I must say I don't understand why the changes in kratos_globals.py are included in this PR: it has nothing to do with JSON and I can't see the new function used within the other changes.

File: kratos/python_scripts/json_output_process.py [hunk 1]
```diff
@@ -120,7 +129,6 @@ def ExecuteFinalizeSolutionStep(self):
                             else:
                                 data["RESULTANT"][out.GetString() ][-1] += value
                     else: # It is a vector
-                        
                         if (KratosMultiphysics.KratosGlobals.HasVariable( out.GetString() + "_X" )): # We will asume to be components
```
Comment (loumalouomega) [inline kratos/python_scripts/json_output_process.py:132]: @jcotela It is here

File: kratos/python_scripts/from_json_check_result_process.py [hunk 2]
```diff
@@ -112,36 +95,36 @@ def ExecuteFinalizeSolutionStep(self):
                             if (self.iscloseavailable == True):
```
Comment (loumalouomega) [inline kratos/python_scripts/from_json_check_result_process.py:95]: @jcotela And here

**Comment by loumalouomega** [comment]
@jcotela The funny thing is that I implemented that Has in the past, but I didn't commit the change, that is why the Has doesn't appear as a change

**Pull request merged**